newpdfgpt/utils/store_data.py

```diff
@@ -0,0 +1,63 @@
+import os
+from langchain_community.document_loaders import DirectoryLoader
+from langchain_text_splitters import CharacterTextSplitter
+from langchain_openai import OpenAIEmbeddings
+from langchain_community.vectorstores import FAISS
+from dotenv import load_dotenv
+from mysql.savefiles import insert_file_name, insert_category
+from mysql import models, database
+
+
+load_dotenv()
```

Comment:
instead of loading dotenv create a constants file create respective constants for all env and use it in all places